components/table/Table.razor

```diff
@@ -159,7 +159,8 @@ RenderFragment<(Table<TItem> table, IEnumerable<TItem> showItems, int level)> bo
             currentRowData.HasChildren = hasChildren;
                 <CascadingValue Value="table.ColumnContext">
                     <CascadingValue Value="currentRowData" Name="RowData">
-                        <tr data-row-id="@rowIndex" class="ant-table-row ant-table-row-level-@level @(currentRowData.Selected ? "ant-table-row-selected" : "")">
+                        <tr @attributes="@(new Dictionary<string,object> { {"onclick", _callbackFactory.Create(table, () => table.RowClick(data)) } })" 
+                             data-row-id="@rowIndex" class="ant-table-row ant-table-row-level-@level @(currentRowData.Selected ? "ant-table-row-selected" : "")">
```

Comment:
I think it is better to export the `RowData` for more information of the row.

Reply:
Tagging @brooklynDev.

Reply:
Done!